scripts/monitoring/cron-send-rpms-versions.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+# Check Docker version
+DOCKER_VERSION=$(docker --version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+HOSTNAME=hostname --fqdn
+
+# Send results to zabbix & hawkular
+ops-metrics-client -s $HOSTNAME -k "docker.version" -o "$DOCKER_VERSION" -t miq_metric="true"
+ops-metrics-client -s $HOSTNAME -k "oc.version" -o "$OC_VERSION" -t miq_metric="true"
```

Comment:
What does miq_metric do?

Does it only send it to manageiq?

I think we could send this to both miq and zbx.